testing/tests/DevExpress.ui.widgets.editors/autocomplete.tests.js

```diff
@@ -1469,6 +1469,45 @@ QUnit.module('regressions', {
         assert.strictEqual(selectionChangedStub.lastCall.args[0].selectedItem, null);
     });
 
+
+    QUnit.module('onSelectionChanged', {
+        beforeEach: function() {
+            this.selectionChangedStub = sinon.stub();
+            this.timeToWait = 300;
+            this.instance.option({
+                searchTimeout: this.timeToWait,
+                onSelectionChanged: this.selectionChangedStub
+            });
+        }
+    }, () => {
+        QUnit.test('onSelectionChanged event should trigger on item selection if its text is equal to input text (T991350)', function(assert) {
+            this.keyboard
+                .type('item 2');
+
+            this.clock.tick(this.timeToWait + 100);
+            const $listItems = $(this.instance.content()).find(`.${LIST_ITEM_CLASS}`);
```

Comment:
```suggestion
            const $listItems = this.instance.content().find(`.${LIST_ITEM_CLASS}`);
```

We'are refactoring all such accessors in a parallel branch now